py-polars/polars/functions/range/datetime_range.py

```diff
@@ -297,6 +297,26 @@ def datetime_ranges(
     -------
     Expr or Series
         Column of data type `List(Datetime)`.
+
+    Examples
+    --------
+    >>> from datetime import datetime
+    >>> df = pl.DataFrame(
+    ...     {
+    ...         "start": [datetime(2022, 1, 1, 0, 0, 0), datetime(2022, 1, 2, 0, 0, 0)],
+    ...         "end": datetime(2022, 1, 3, 0, 0, 0),
+    ...     }
+    ... )
+    >>> df.select(date_range=pl.datetime_ranges("start", "end"))  # doctest: +SKIP
+    shape: (2, 1)
+    ┌─────────────────────────────────────────────────────────────────┐
+    │ date_range                                                      │
+    │ ---                                                             │
+    │ list[datetime[μs]]                                              │
+    ╞═════════════════════════════════════════════════════════════════╡
+    │ [2022-01-01 00:00:00, 2022-01-02 00:00:00, 2022-01-03 00:00:00] │
+    │ [2022-01-02 00:00:00, 2022-01-03 00:00:00]                      │
+    └─────────────────────────────────────────────────────────────────┘
```

Comment:
```suggestion
>>> with pl.Config(fmt_str_lengths=65):
...    print(df.select(date_range=pl.datetime_ranges("start", "end")))
shape: (2, 1)
┌─────────────────────────────────────────────────────────────────┐
│ date_range                                                      │
│ ---                                                             │
│ list[datetime[μs]]                                              │
╞═════════════════════════════════════════════════════════════════╡
│ [2022-01-01 00:00:00, 2022-01-02 00:00:00, 2022-01-03 00:00:00] │
│ [2022-01-02 00:00:00, 2022-01-03 00:00:00]                      │
└─────────────────────────────────────────────────────────────────┘
```

Reply:
> Suggestions?

Try the above; demonstrates ad-hoc use of `Config` and allows the full width to display 👌

Reply:
OK, cool. Thanks!